src/Evolution/DiscontinuousGalerkin/SlopeLimiters/Minmod.hpp

```diff
@@ -0,0 +1,309 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <type_traits>
+#include <unordered_map>
+#include <utility>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Element.hpp"  // IWYU pragma: keep
+#include "ErrorHandling/Assert.hpp"
+#include "NumericalAlgorithms/LinearOperators/MeanValue.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/ForceInline.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+template <size_t VolumeDim>
+class Direction;
+template <size_t>
+class Mesh;
+
+namespace PUP {
+class er;
+}  // namespace PUP
+
+namespace SlopeLimiters {
+template <size_t VolumeDim, typename TagsToLimit>
+class Minmod;
+}  // namespace SlopeLimiters
+/// \endcond
+
+namespace SlopeLimiters {
+/// \ingroup SlopeLimitersGroup
+/// \brief Possible types of the minmod slope limiter.
+///
+/// \see The documentation of `Minmod`, where these types are discussed.
```

Comment:
"Minmod" isn't showing up as a link in Doxygen

Reply:
:( even with the namespace prefixed? I know it uses the C++ identifier name

Reply:
Of the things I've tried, the only one that was successful was to move the Minmod forward decl out of `\cond`.

Reply:
So I left as is (dox unlinked), for consistency of `cond` with rest of code.